### PR TITLE
[Communication] - Identifiers - Arch review board adding identifiers suffix

### PR DESCRIPTION
**Pull request by JoshuaLai**

### CODE REVIEW

File: examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ class BlobDownloadViewController: UIViewController, MSALInteractiveDelegate {
 
     @IBOutlet var tableView: UITableView!
 
-    private lazy var player: AVPlayer = AVPlayer()
+    private lazy var player = AVPlayer()
```
Comment (JoshuaLai) [inline examples/AzureSDKDemoSwift/Source/BlobDownloadViewController.swift:43]: Change was introduced by our linter

File: sdk/core/AzureCore/Source/Pipeline/Policies/LoggingPolicy.swift [hunk 2]
```diff
@@ -178,19 +178,22 @@ public class LoggingPolicy: PipelineStage {
     private func humanReadable(body bodyFunc: () -> String?, headers: HTTPHeaders) -> String {
         if
             let encoding = headers[.contentEncoding],
-            encoding != "" && encoding.caseInsensitiveCompare("identity") != .orderedSame {
+            encoding != "" && encoding.caseInsensitiveCompare("identity") != .orderedSame
```
Comment (JoshuaLai) [inline sdk/core/AzureCore/Source/Pipeline/Policies/LoggingPolicy.swift:181]: Change was introduced by our linter

**Comment by JoshuaLai** [comment]
@minnieliu @DominikMe Pr for changes that include the `identifiers` suffix

**Review by minnieliu — APPROVED** [review]
LGTM!

**Pull request merged**